bin/templates/project/app/build.gradle

```diff
@@ -30,13 +32,14 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
Can we wrap all of the Kotlin code and make it conditional by a flag? This is regarding to the same concerns that @dpogue mentioned.

It might be possible that not everyone would use Kotlin.

For example:
```
if (cdvHelpers.getConfigPreference('EnableKotlin', 'false').toBoolean()) {
  classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
}
```

Reply:
And this conditional statement would be applied everywhere else relating to Kotlin.

Reply:
Awesome, did not know this was available.  Thanks

Reply:
@joshchandler Sorry, the `cdvHelpers` is currently not available in the `dependencies` scope, as I mentioned in this example. 

Wrapping the apply plugin should be OK. Example:
```
if (cdvHelpers.getConfigPreference('EnableKotlin', 'false').toBoolean()) {
    apply plugin: 'kotlin-android'
}
```

There is a discussion on a PR that also modifies the the same `build.gradle` file for other features. It also had the same issue where `cdvHelpers` is not available in the dependency scope. Adding it is as easy as adding the `apply from: '../CordovaLib/cordova.gradle'` in the scope BUT I am waiting for any feedback about it.  
You can see the comment here: https://github.com/apache/cordova-android/pull/438#discussion_r216195552

Reply:
Oh I see.. Ok well then I'll wait until feedback is received on that google services PR.  I'm not familiar enough with gradle to confidently say that importing into the dependencies scope wouldn't cause any problems.  `cordova.gradle` only consists of helper methods, so I don't see any reason why it would be an issue though.

If it doesn't cause any issues then this would just be another reason to pull it in here and the other dependencies scope further below this file.